nameable.rb

```diff
@@ -0,0 +1,37 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError, 'Method not implemented'
+  end
+end
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super
```

Comment:
- [ ] The same happens with this one! Please consider adding parenthesis. 🤓 🔧 

```suggestion
    super()
```